docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,89 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
```

Comment:
There's also the Windows-specific image-recognition test that Etcher does, which we should probably test

Reply:
Sounds great.